libr/bin/format/elf/elf.c

```diff
@@ -1420,6 +1420,34 @@ static size_t get_num_relocs_dynamic_plt(ELFOBJ *bin) {
 	return 0;
 }
 
+static ut64 get_import_addr_riscv(ELFOBJ *bin, RBinElfReloc *rel) {
```

Comment:
@08A you do like to make my life harder by moving code around? :D Joking, but on a more serious note, please avoid moving too many stuff when you are doing a big PR, as that really makes it harder to read the diff.

Reply:
Sorry it was just that the function wasn't in alphabetical order.